### PR TITLE
checkbox fix#1673

### LINKED ISSUE #1763: Complete div behaves as checkbox rather than just the checkbox on Login Page
**Describe the bug**
Clicking the whole div selects/deselects the checkbox.

**To Reproduce**
1. Go to [CircuitVerse Login Page](https://circuitverse.org/users/sign_in)
2. Click on and around the 'Remember Me' Checkbox
3. It selects/deselects the Remember Me Checkbox

**Expected behavior**
Only clicking the checkbox should select/deselect it.

**Screenshots**
image

**Are you working on this issue? (Yes/No)**
Nope.

### PR DESCRIPTION
**Pull request by pavanjoshi914**
Fixes #1763 

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by satu0king** [comment]
Looks good to me!

**Review by satu0king — APPROVED** [review]

**Comment by satu0king** [comment]
@aayush-05 check this one

File: app/assets/stylesheets/components.scss [hunk 1]
```diff
@@ -251,9 +251,11 @@
 }
 
 .primary-checkpoint-container {
+  bottom: 1rem;
   cursor: pointer;
-  display: block;
+  display: inline-block;
   font-size: 22px;
+  left: 7rem;
```
Comment (aayush-05) [inline app/assets/stylesheets/components.scss:258]: Why was this required? If you require it to align the checkbox better; please add a separate class to this particular checkbox and move the style there. It can be saved in app/assets/stylesheets/users.scss
Changing this on main checkbox class would break other ones on platform.

File: app/views/users/sessions/new.html.erb [hunk 2]
```diff
@@ -36,7 +36,8 @@
         </div>
         <% if devise_mapping.rememberable? -%>
           <div class="form-group newproject-checkbox">
-            <label class="primary-checkpoint-container users-primary-checkpoint"><h6>Remember Me</h6>
+           <h6>Remember Me</h6>
+            <label class="primary-checkpoint-container users-primary-checkpoint">
```
Comment (aayush-05) [inline app/views/users/sessions/new.html.erb:40]: This change isn't required. We want the checkbox to be accessible via the Label text.
Rest all LGTM

File: app/assets/stylesheets/components.scss (hunk 1 above)
Comment (pavanjoshi914) [reply app/assets/stylesheets/components.scss:258]: because i moved h1 above label somehow checkbox lost its alignment so to make it align done this

File: app/views/users/sessions/new.html.erb (hunk 2 above)
Comment (pavanjoshi914) [reply app/views/users/sessions/new.html.erb:40]: It was mentioned in the issue that checkbox should be accessed only through checkbox so according to mentions in issue 
I excluded h1
if  we allow then only change we need is display:inline-block

Comment (aayush-05) [reply app/views/users/sessions/new.html.erb:40]: Will confirm this and then we can move ahead

Comment (satu0king) [reply app/views/users/sessions/new.html.erb:40]: > Will confirm this and then we can move ahead

Any update?

Comment (aayush-05) [reply app/views/users/sessions/new.html.erb:40]: Having it accessible from the label text is a good option. Let us go with that @pavanjoshi914 . The fix you mentioned would do

Comment (pavanjoshi914) [reply app/views/users/sessions/new.html.erb:40]: Ohk implementing following changes

**Comment by pavanjoshi914** [comment]
after discussion checkbox will we accessible through the label... change made in #2029
